### DISCUSSION TITLE
About the parameters for generating grains based on a pdf

### DISCUSSION THREAD

**Discussion by mario-renderos** [neperfepx/neper#919] [Q&A] [answered]
Hello, good morning.

I have a question about the parameters when using probability distribution functions to generate grains shape. For the skewnormal distribution there are three parameters (mean, sigma and gamma).
I am clear that sigma is the square root of the variance of X (sigma=sqrt(variance)), mean is the first moment of X (mean = E(X)). What I am not clear about is what the gamma parameter corresponds to. Is gamma the skewness (generally denoted as gamma_1), or is it the kurtosis (usually denoted as gamma_2)?

Thank you very much for the help, Mario.

**Comment by rquey** [comment, marked as answer]
I don't know (this distribution was contributed), but here is the code:
https://github.com/neperfepx/neper/blob/27841d7c78670b7a916b38b3406723c96270ed7e/src/contrib/ut/src/ut_fct/ut_fct.c#L468-L477

**Comment by mario-renderos** [comment]
Hello.
There is an error in the way the parameters for the skewnormal probability density function are defined.
In the manual:
- mu appears as the mean, it is actually the location factor (xi).
- sigma as the standard deviation, but it is the scale factor (omega).
- gamma, is supposed to be sigma?, but it is actually the shape factor (alpha).
https://en.wikipedia.org/wiki/Skew_normal_distribution

It would be interesting to correct that in the manual, otherwise very important errors are generated.

Kind regards, Mario.
